examples/javascript/test/browser/chromeSpecificCaps.spec.js

```diff
@@ -47,6 +47,17 @@ suite(function (env) {
       await driver.quit();
     });
 
+    it('Add Extension', async function () {
+      const options = new Chrome.Options();
+      let driver = await env
+        .builder()
+        .setChromeOptions(options.addExtensions(['examples/javascript/test/extensions/webextensions-selenium-example.crx']))
```

Comment:
HI @by-gayathri ,

Thank you for the changes!


direct path is not going to work, you need to use node `path` to resolve path!

This can help https://github.com/SeleniumHQ/seleniumhq.github.io/blob/trunk//examples/javascript/test/browser/firefoxSpecificFunctionalities.js#L22-L24

Reply:
That was an interim one. Pushed all changes now. Ran the tests in local to verify. Verified in hugo server too. Please review

Reply:
Addressed @titusfortner 's comments too